### DISCUSSION TITLE
Getting help with open source project

### DISCUSSION THREAD

**Discussion by ghost** [community/community#69322] [Programming Help]
### Body

Hi,

I am a student who is learning how to code and I am creating a website that will allow users to change files to links so, students can submit work more easily.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by moayad-star** [comment]
Great work, but have you thought about the amount of storage space you need? It is an expensive and difficult matter that cannot be done by one or more programmers, but rather requires an integrated team to appear as required.
You can change the idea to something that does not consume money like this, and your capital will certainly not be enough
However, there is a brilliant way for the project to succeed, which is to make each file have a validity period that does not exceed a month and then the file is deleted. If the user wants more time, he can pay a small amount.
